ci/no-std-check/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
```

Comment:
Probably a leftover from the merge, we can remove this.

Reply:
[f87e34d](https://github.com/informalsystems/hermes/pull/2873/commits/f87e34de7e4336f47b33696660404609c99f9e98)